{"uuid": "9238bc75-5e09-4790-8c8c-2ffc9326b831", "children": ["0ee771c2-d447-4c7a-b0e1-8620b4060010"], "befores": [{"name": "driver_class", "status": "broken", "statusDetails": {"message": "_pytest.config.exceptions.UsageError: --driver must be specified\n", "trace": "  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\pluggy\\_callers.py\", line 39, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 1126, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\_pytest\\fixtures.py\", line 932, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n  File \"D:\\work_projects\\venvs\\keka_automation\\lib\\site-packages\\pytest_selenium\\pytest_selenium.py\", line 181, in driver_class\n    raise pytest.UsageError(\"--driver must be specified\")\n"}, "start": 1678341015387, "stop": 1678341015387}], "start": 1678341015387, "stop": 1678341081452}